dash/development/_all_keywords.py

```diff
@@ -1,15 +1,6 @@
-# This is a set of Python keywords that cannot be used as prop names.
-# Keywords for a particular version are obtained as follows:
-# >>> import keyword
-# >>> keyword.kwlist
+import keyword
 
-python_keywords = {
-    'and', 'elif', 'is', 'global', 'as', 'in', 'if', 'from', 'raise', 'for',
-    'except', 'nonlocal', 'pass', 'finally', 'print', 'import', 'True', 'None',
-    'return', 'exec', 'await', 'else', 'break', 'not', 'with', 'class',
-    'assert', 'False', 'yield', 'try', 'while', 'continue', 'del', 'async',
-    'or', 'def', 'lambda'
-}
+python_keywords = set(keyword.kwlist)
```

Comment:
python 2 and 3 have a different keyword set

Reply:
I think we want the union of keywords for both python version here - since this is used in component generation and we want generated components to work in both 2 and 3 regardless of which version they're built with.

In addition seems to me we should prevent building components using keywords as prop names - currently they just get dropped from the prop list. Or perhaps prefix keyword props with something so they could still be used?

Reply:
yes, it makes sense to have union here. good point!

the only usage in the code is here https://github.com/plotly/dash/blob/076f5ec92b9bd034d3904e97bd58fc6207482886/dash/development/_py_components_generation.py#L90-L99

Reply:
Broke out the "in addition" point into https://github.com/plotly/dash/issues/878
For now let's just put back the union keyword list.